tests/spec/EntryCollector.spec.php

```
@@ -0,0 +1,90 @@
+use Psr\Container\ContainerInterface;
+
+describe(EntryCollector::class, function () {
+    given('collector', function () {
+        return new EntryCollector();
+    });
+
+    it('should be able to set and get an entry', function () {
+        $this->collector['foo'] = 'bar';
+
+        expect($this->collector['foo'])->toBe('bar');
+        expect(isset($this->collector['foo']))->toBeTruthy();
```

PHPStan (2.1.54)

[error] Variable $this in isset() is never defined.


(isset.variable)